.github/actions/install-briefcase/action.yml

```diff
@@ -0,0 +1,101 @@
+name: Install Briefcase
+description: Installs a version of Briefcase appropriate for the context.
+
+inputs:
+  briefcase-url:
+    description: 'URL to Briefcase git repository.'
```

Comment:
We've been trying to converge on " rather than ' in config files (for broad consistency with black); is there a particular reason you've reverted a bunch of those changes and moved to '?

Reply:
Mostly because `"` and `'` have different functional meanings in YAML....while in Python they are functionally equivalent.

Double quoted strings in YAML pass through a level of escape parsing that single quotes strings do not. So, really, I'm trying to emphasize that I literally want the value between the two single quotes without any interpretation.

That said, none of this is particularly consistent. For instance, all the places that GitHub is substituting values for `${{ github.event_name }}` (and the like), practically none of those are quoted at all.

So, unless this is compelling, I'm fine converging on double quotes. I also just realized that I was confusing TOML with YAML; TOML is a bit more restrictive about double quotes...